src/System.IO.FileSystem/tests/Directory/CreateDirectory.cs

```diff
@@ -206,19 +206,6 @@ public void DirectoryEqualToMaxDirectory_CanBeCreatedAllAtOnce()
             Assert.True(Directory.Exists(result.FullName));
         }
 
-        [Fact]
-        public void DirectoryWithComponentLongerThanMaxComponentAsPath_ThrowsPathTooLongException()
-        {
-            // While paths themselves can be up to 260 characters including trailing null, file systems
-            // limit each components of the path to a total of 255 characters.
-            var paths = IOInputs.GetPathsWithComponentLongerThanMaxComponent();
-
-            Assert.All(paths, (path) =>
-            {
-                Assert.Throws<PathTooLongException>(() => Create(path));
-            });
-        }
-
```

Comment:
That means we should have tests growing the path length until an IOException happens, right?